### PR TITLE
Mzk/ceramic

### PR DESCRIPTION
**Pull request by mzkrasner**
# Description

This PR reflects the new "Ceramic Example" page added to show how to store encrypted data (using LIT) on ComposeDB (uses LIT SDK v3)

## Type of change

Please delete options that are not relevant.

- [x ] Introducing new feature (non-breaking change which adds functionality)
- (not necessarily a new feature, but rather a tutorial to show how the two technologies can complement each other)

General
- [x ] I have performed a self-review of my code
- [x ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ x] I have checked the additions are concise
- [ x] Language is consistent with existing documentation
- [ x] My changes generate no new warnings
- [x ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ x] included a beginner friendly explanation
- [ x] included a basic technical introduction and code sample
- [x ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Comment by mzkrasner** [comment]
direct preview link: https://deploy-preview-144--lit-dev-docs.netlify.app/v3/resources/ceramicExample

File: docs/integrations/ceramicExample.md [hunk 1]
```diff
@@ -16,21 +16,23 @@ Learn how to use Lit Protocol to encrypt messages to yourself and save them to t
...
 
 For the purpose of this tutorial, we will be using Ceramic with ComposeDB to illustrate how developers can generate, store, and query simple encrypted messages.
 
-This tutorial will use a message board example application to show how to create encrypted messages using Lit Protocol and save message instances to the Ceramic Network using ComposeDB, and will reference this [example repository](https://github.com/mzkrasner/lit-composedb).
+This tutorial will use a message board example application to show how to create encrypted messages using Lit Protocol and save message instances to the Ceramic Network using ComposeDB.
+
+To follow along, reference this [example repository](https://github.com/mzkrasner/lit-composedb).
```
Comment (mzkrasner) [inline docs/integrations/ceramicExample.md:35]: hello @debbly ! I'm glad you caught this - this should actually be https://github.com/ceramicstudio/lit-composedb (forgot to update it)

**Review by mzkrasner — COMMENTED** [review]
the new location looks great!

File: docs/integrations/ceramicExample.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
+---
+sidebar_position: 3
+---
+
+# ComposeDB (Ceramic)
```
Comment (debbly) [inline docs/integrations/ceramicExample.md:5]: Ideally we have a table here with objectives for what this page will cover! 

In some docs, it's in the format:

```
## Objectives
At completion of this reading you should be able to:

- Explain at a high level what encryption and programmatic signing is.
```

File: docs/integrations/ceramicExample.md (hunk 1 above)
Comment (debbly) [reply docs/integrations/ceramicExample.md:35]: Awesome, can you update it in the file?

**Comment by debbly** [comment]
Accepted pr with these fixes: https://github.com/LIT-Protocol/docs/pull/146

**Pull request closed without merging**